com.google.devtools.bazel.e4b/src/com/google/devtools/bazel/e4b/preferences/BazelPreferenceInitializer.java

```diff
@@ -25,10 +28,24 @@
  */
 public class BazelPreferenceInitializer extends AbstractPreferenceInitializer {
 
+  private static final ImmutableList<String> BAZEL_PATH_CANDIDATES =
+      ImmutableList.of("/usr/local/bin/bazel", "/usr/bin/bazel");
+
   @Override
   public void initializeDefaultPreferences() {
     IPreferenceStore store = Activator.getDefault().getPreferenceStore();
-    store.setDefault("BAZEL_PATH", Activator.DEFAULT_BAZEL_PATH);
+    store.setDefault("BAZEL_PATH", findBazelPath());
   }
 
+  private static String findBazelPath() {
+    for (String path : BAZEL_PATH_CANDIDATES) {
+      if (new File(path).isFile()) {
+        return path;
+      }
+    }
+
+    System.err.println("No candidate path for blaze found. Using default.");
```

Comment:
Also maybe tell in the message to go to the Eclipse preferences > Bazel to provide the path to Bazel.